edi_storage_oca/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 ACSONE
+# @author: Simone Orsi <simahawk@gmail.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "EDI Storage backend support",
+    "summary": """
+    Base module to allow exchanging files via storage backend (eg: SFTP).
+    """,
+    "version": "18.0.1.0.0",
+    "development_status": "Beta",
+    "license": "LGPL-3",
+    "website": "https://github.com/OCA/edi-framework",
+    "author": "ACSONE,Odoo Community Association (OCA)",
+    "depends": ["edi_oca", "fs_storage", "component"],
```

Comment:
@simahawk @etobella Related with my question here https://github.com/OCA/storage/issues/251#issuecomment-3273671218. If both fs_storage and storage_backend will continue to coexist... doesn't it make sense to have edi_fs_storage and edi_storage? Thoughts? 

PD: Again if this has been already discussed, I apologize.

Reply:
There are migration scripts that transform from storage to fs_storage. It was decided on 16 that the new connection protocol in EDI was fs_storage. The migration scripts are there